apps/web/pages/event-types/index.tsx

```diff
@@ -925,12 +925,12 @@ const EventTypesPage = () => {
 
                 {data.eventTypeGroups.length === 0 && <CreateFirstEventTypeView />}
 
-                <EmbedDialog />
                 {router.query.dialog === "duplicate" && <DuplicateDialog />}
               </>
             );
           }}
         />
+        <EmbedDialog />
```

Comment:
This solves the problem of the popup appearing and disappearing.

In `apps/web/components/eventtype/EventTypeSingleLayout.tsx`

The component is a direct child of `Shell`, and the problem does not happen on the single event page.